### PR TITLE
RCT/YJ/get_swh_uses_associated_with_each_building_segment

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    ----------
+    rmd: dict
+        RMD at RuleSetModelDescription level
+    building_segment_id: str
+        building segment id
+
+    Returns
+    -------
+    swh_uses: list
+        A list containing the ids of all service water heating uses associated with a building segment
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py:21]: Add additional to include extreme case ` or an empty list if no service water heating uses are found in the building segment.`

**Review by weilixu — APPROVED** [review]

**Comment by weilixu** [comment]
Approved with one minor update request.
Good work, thank you Yun Joon.

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py:21]: Addressed.

**Review by Jiarongx-Xie — APPROVED** [review]
Looks good to me. Thank you Yun Joon!

**Pull request merged**